code-for-canada/project-thundercat
# [How To Page] Building the Test Instructions Section

# Description

Added the content of Test Instructions in its respective component and format everything.

## Type of change

- New feature (non-breaking change which adds functionality)
- Code cleanliness or refactor

## Screenshot

**Important Section Added [ENGLISH]:**

![image](https://user-images.githubusercontent.com/23021242/53243897-4d748200-3677-11e9-9a0e-1ac45e2afa08.png)
![image](https://user-images.githubusercontent.com/23021242/53243915-57968080-3677-11e9-84bf-76831aaaa39c.png)
![image](https://user-images.githubusercontent.com/23021242/53243925-60875200-3677-11e9-98e8-1999d02dea34.png)
![image](https://user-images.githubusercontent.com/23021242/53243954-71d05e80-3677-11e9-8350-51346349297f.png)
![image](https://user-images.githubusercontent.com/23021242/53243969-7f85e400-3677-11e9-82f8-3850c6fa24a6.png)
![image](https://user-images.githubusercontent.com/23021242/53243980-86145b80-3677-11e9-8875-92e29bf3e98a.png)

**Important Section Added [FRENCH]:**

![image](https://user-images.githubusercontent.com/23021242/53244024-a04e3980-3677-11e9-9c68-5d0c4e1e0b20.png)
![image](https://user-images.githubusercontent.com/23021242/53244037-a93f0b00-3677-11e9-8c81-0791e4ef8a7e.png)
![image](https://user-images.githubusercontent.com/23021242/53244077-c07df880-3677-11e9-8517-30eb6f7b18a9.png)
![image](https://user-images.githubusercontent.com/23021242/53244086-c83d9d00-3677-11e9-9035-2804c22b7118.png)
![image](https://user-images.githubusercontent.com/23021242/53244094-d2f83200-3677-11e9-83cf-86dc44e63d14.png)
![image](https://user-images.githubusercontent.com/23021242/53244101-d986a980-3677-11e9-8996-4d3bfcd1ae3a.png)

**NOTE THAT THE IMAGES ARE NOT TRANSLATED IN FRENCH YET. THIS WILL BE FIXED IN ANOTHER PR**

# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1. Go to Prototype tab.
2. Click _Start eMIB Sample Test_.
3. Click _Next_.
4. Make sure that Test Instructions section looks good and is the same as the wireframes (except the content itself).

Screenshot of unit tests run passing:

![image](https://user-images.githubusercontent.com/23021242/53244414-d04a0c80-3678-11e9-8094-ef5d127d6b53.png)

# Checklist

Applicable for all code changes.
